app/controllers/invitations_controller.rb

```diff
@@ -39,7 +39,8 @@ def create
       InvitationService.invite_to_group(recipient_emails: @invite_people_form.emails_to_invite,
                                         message: @invite_people_form.message_body,
                                         group: @invitable,
-                                        inviter: current_user)
+                                        inviter: current_user,
+                                        subject: params[:invite_people_form][:subject])
```

Comment:
ehr... yes, i'll strip spaces and new lines and put a limit, maybe 100 characters? (doesn't the mail gem take care of this?)